### PR TITLE
Add Jupyterlite Terminal blog post.

### PR DESCRIPTION
**Pull request by HaudinFlorence**
Add Jupyterlite Terminal blog post.

### CODE REVIEW

**Review by ianthomas23 — APPROVED** [review]
@HaudinFlorence Thanks for doing this. It all looks good to me.

File: src/components/blog/blogpostsDetails.json [hunk 1]
```diff
@@ -1,4 +1,17 @@
 [
+  {
+  "url": "https://blog.jupyter.org/jupyterlite-terminal-edb3f80dc1c0",
+  "title": "Jupyterlite Terminal",
```
Comment (jtpio) [inline src/components/blog/blogpostsDetails.json:4]:
```suggestion
  "title": "JupyterLite Terminal",
```

Comment (jtpio) [reply src/components/blog/blogpostsDetails.json:4]: Minor nit-pick :)

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**